### PR TITLE
l10n: zh-TW: update to 0.16.5

### PR DESCRIPTION
**Pull request by 5idereal**

### CODE REVIEW

**Comment by 5idereal** [comment]
@pan93412 @emisjerry

**Comment by pan93412** [comment]
I'll review it tomorrow.

File: zh-TW.json [hunk 1]
```diff
@@ -159,8 +159,8 @@
 			"option-manage-themes-description": "管理已安裝主題及瀏覽社群主題。",
 			"option-theme-button-manage": "管理",
 			"option-choice-none": "無",
-			"option-native-menus": "Native menus",
-			"option-native-menus-desc": "Menus throughout the app will match the operating system. They will not be affected by your theme.",
+			"option-native-menus": "原生選單",
+			"option-native-menus-desc": "整個應用程式的選單將與作業系統相符。並不會受您的主題影響。",
```
Comment (pan93412) [inline zh-TW.json:163]: 整個應用程式的選單將配合作業系統的風格，不受您的主題影響。

File: zh-TW.json [hunk 2]
```diff
@@ -473,7 +473,7 @@
 		"msg-upgrade-installer": "欲使用此功能，請由官網下載，並重新安裝最新版本的應用程式。",
 		"tooltip-restore-default-settings": "還原為預設設定",
 		"label-copy": "複製到系統剪貼簿",
-		"copied_generic": "Copied to your clipboard",
+		"copied_generic": "已複製到您的剪貼簿",
```
Comment (pan93412) [inline zh-TW.json:476]: `已複製到剪貼簿` should be enough.

File: zh-TW.json [hunk 3]
```diff
@@ -505,28 +505,28 @@
...
+			"close-others": "關閉其他",
+			"close-right": "關閉右側分頁",
 			"pin": "釘選",
 			"unpin": "取消釘選",
-			"unlink-tab": "Unlink tab",
-			"link-tab": "Link with tab...",
-			"toggle-source-mode": "Source mode",
-			"toggle-reading-view": "Reading view",
+			"unlink-tab": "取消連結分頁",
+			"link-tab": "與分頁連結...",
```
Comment (pan93412) [inline zh-TW.json:520]: Use Unicode punctuation: `…`

File: zh-TW.json [hunk 4]
```diff
@@ -559,7 +559,7 @@
 			"option-new-vault-location-description": "選擇置放儲存庫的位置。",
 			"label-new-vault-location-preview": "您的新儲存庫會放在：",
 			"option-reveal-vault-in-explorer": "在系統檔案總管中開啟儲存庫",
-			"option-reveal-vault-in-explorer-mac": "Reveal vault in Finder",
+			"option-reveal-vault-in-explorer-mac": "在 Finder 中顯示儲存庫",
 			"option-rename-vault": "重新命名儲存庫……",
```
Comment (pan93412) [inline zh-TW.json:563]: Should include only a `…`, as this is an indicator to next operation.

File: zh-TW.json [hunk 5]
```diff
@@ -536,8 +536,8 @@
 			"rename-blockid": "重新命名此區塊 ID……",
```
Comment (pan93412) [inline zh-TW.json:536]: Should include only a `…`, as this is an indicator to next operation.

File: zh-TW.json [hunk 6]
```diff
@@ -774,18 +774,18 @@
...
 		"fold-all": "全部收合",
 		"unfold-all": "全部展開",
-		"fold-more": "Fold More",
-		"fold-less": "Fold Less",
-		"source-mode": "Source Mode",
-		"reading-view": "Reading View",
+		"fold-more": "收合更多",
+		"fold-less": "收合較少",
+		"source-mode": "原始碼模式",
+		"reading-view": "閱讀檢視",
```
Comment (pan93412) [inline zh-TW.json:780]: `閱讀顯示模式` ？

File: zh-TW.json [hunk 7]
```diff
@@ -795,9 +795,9 @@
 			"action-open": "開啟檔案瀏覽器",
 			"action-show": "顯示檔案瀏覽器",
 			"action-create-note": "新增筆記",
-			"action-create-note-to-the-right": "Create note to the right",
-			"action-collapse-all": "Collapse all",
-			"action-expand-all": "Expand all",
+			"action-create-note-to-the-right": "在右側新增筆記",
```
Comment (pan93412) [inline zh-TW.json:798]: `在右側建立筆記`: `新增` is preferred to be the translation of `New`.

File: zh-TW.json [hunk 8]
```diff
@@ -1040,28 +1040,28 @@
...
-			"msg-successfully-installed-theme": "Successfully installed theme \"{{name}}\".",
-			"msg-installing-theme": "Installing theme \"{{name}}\"...",
-			"label-search-summary": "Showing {{themeCount}}:",
-			"label-update-available": "Update available"
+			"label-installed": "已安裝",
+			"label-legacy": "舊版",
+			"button-update-all-themes": "全部更新",
+			"msg-failed-load-themes": "無法載入社群主題。",
+			"msg-no-updates-found": "找不到可用的主題更新。",
+			"msg-updates-found": "{{count}} 個主題可更新。",
```
Comment (pan93412) [inline zh-TW.json:1058]: `Found` can be translated as `有`: `有 {{count}} 個主題可更新。`

File: zh-TW.json [hunk 9]
```diff
@@ -1040,28 +1040,28 @@
...
+			"msg-updates-found_plural": "{{count}} 個主題可更新。",
+			"msg-failed-to-install-theme": "無法安裝主題 \"{{name}}\"。",
+			"msg-successfully-installed-theme": "成功安裝主題 \"{{name}}\"。",
+			"msg-installing-theme": "正在安裝主題 \"{{name}}\"……",
```
Comment (pan93412) [inline zh-TW.json:1062]: The current consensus is `「內容」` instead of `"內容"`. This is discussable.

File: zh-TW.json [hunk 10]
```diff
@@ -1553,10 +1553,10 @@
...
 			"label-not-sharing": "此遠端儲存庫目前未與任何人共享。",
 			"label-vaults-shared-with-you": "與您共享的儲存庫",
-			"tooltip-leave-vault-sharing": "Stop collaborating on this vault",
-			"label-leave-vault-confirmation": "Confirm stop vault collaboration",
-			"label-leave-vault-confirmation-details": "This will remove this vault from the list of vaults shared with you. This action cannot be reverted.",
-			"label-leave-vault-confirmation-details-2": "Please contact the owner of the vault if you wish to collaborate on this vault again.",
+			"tooltip-leave-vault-sharing": "停止於此儲存庫協作",
+			"label-leave-vault-confirmation": "確認停止儲存庫協作",
+			"label-leave-vault-confirmation-details": "這將從與您共享的儲存庫列表中移除此儲存庫。此操作無法復原。",
+			"label-leave-vault-confirmation-details-2": "若您希望再次於此儲存庫協作，請聯繫儲存庫的擁有者。",
```
Comment (pan93412) [inline zh-TW.json:1559]: `聯絡` instead of `聯繫`

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: zh-TW.json [hunk 11]
```diff
@@ -533,7 +533,7 @@
 			"paste-as-plain-text": "以純文字貼上",
 			"select-all": "全選",
 			"rename-heading": "重新命名標題……",
```
Comment (pan93412) [inline zh-TW.json:535]: I just found another `…` issue. 😢

**Review by emisjerry — APPROVED** [review]

**Review by pan93412 — APPROVED** [review]

**Pull request merged**